samples/msal-node-samples/ElectronSystemBrowserApp/README.md

```diff
@@ -0,0 +1,204 @@
+
+import  * as APP_CONFIG from "./config/YOUR_CUSTOM_CONFIG_FILE.json"; // To this
+```
+
+This application uses the `User.Read` and `Mail.Read` Microsoft Graph Scopes, so make sure they are enabled in your App Registration.
+
+**Note: If you'd like to configure custom scopes for this sample application, you'll need to modify the request scopes used in the `setRequestObjects()` method within `AuthProvider.ts`.**
+
+#### Registering the application
+
+Whether or not you decide to customize these values, you must register the `redirectUri` in the Azure Portal as a Mobile or Desktop Redirect URI.
+
+1. Go to the App Registration in the Azure Portal.
+2. Click the `Authentication` tab in the side menu.
+3. Under "Platform configurations", click the "Add a platform" link.
+4. Select "Mobile and desktop applications".
+5. In the Redirect URI section,  enter the following redirect URI `http://localhost:8080`.
+6. Copy the exact `redirectUri` value (`msal4b0db8c2-9f26-4417-8bde-3f0e3656f8e0://auth` we will use this as our custom URL scheme in the web page to link to our electron app.
+7. Click "Configure"
+
+### Executing the application
+
+Once you are in the sample application directory and you've configured the sample application to match your App Registration and registered Redirect URI, you can install all of the dependencies by running:
+
+```bash
+npm install
+```
+
+When the dependencies have been installed, you can run the sample application by using the following command, after which the Electron application should start.
+
+```bash
+cd samples/msal-node-samples/standalone-samples/ElectronSystemBrowserApp/redirect
+$ npm start
+```
+
+```bash
+cd samples/msal-node-samples/standalone-samples/ElectronSystemBrowserApp
+npm start
+```
+
+## About the code
+
+### Custom URL Scheme
+
+To demonstrate best security practices, this Electron sample uses a custom URL scheme `msal4b0db8c2-9f26-4417-8bde-3f0e3656f8e0://auth` that will launch the app when the URL with that scheme is visited.
+
+Using this method, the desktop application starts the [Authorization code flow with PKCE flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) as normal by lunching the system browser, authenticating the user, and redirecting with the authorization code in the browser.
+
+On receiving the authorization code from the **AzureAD**, the browser will launch the application with the custom URL scheme and the code as query parameters. The desktop will parse the authorization code to authenticate the user with **MSAL**.
+
+#### Redirect page
+
+We used a node server to host a static web page `.\redirect\index.html` to serve as a redirect page. The page will receive the authorization code as query parameter from **AzureAD** and on button click will trigger the Custom URL Scheme `msal4b0db8c2-9f26-4417-8bde-3f0e3656f8e0://auth` to forward the authorization code to the electron application.
```

Comment:
Perhaps we can mention hosting this page on Azure Storage static websites or something similar?